### DISCUSSION TITLE
Quickdraw badge on profile doesn't flip

### DISCUSSION THREAD

**Discussion by Shide** [community/community#38257] [Other Feature Feedback, Questions, & Ideas] [answered]
Quickdraw badge doesn't flip on mouse hover (profile page)

**Comment by jfullstackdev** [comment]
Pull Shark is flipping but not Quickdraw on my end 

and Pull Shark is the only one flipping now

**Comment by radar-blip** [comment, marked as answer]
I dont think it supposed to, is it? 
Thats just a default badge, it doesn't have different levels (x2, x3).

**Comment by fcasamento** [comment]
@Shide ,  As our colleague @linuxlawson  mentioned this emblem has no levels.
Here's a link with the details of what you mentioned and the other emblems.

https://github.com/Schweinepriester/github-profile-achievements
